packages/site-kit/src/lib/styles/tokens/colours.css

```diff
@@ -0,0 +1,108 @@
+/*
+
+!IMPORTANT: The tokens for light and dark mode are duplicated, if you change a token for dark mode,
+!you should change that for the other dark mode too
+
+*/
+
+:root {
+	color-scheme: light;
+
+	/* Foreground colours */
+	--sk-fg-1: hsl(0, 0%, 5%);
+	--sk-fg-2: hsl(0, 0%, 12%);
+	--sk-fg-3: hsl(0, 0%, 27%);
+	--sk-fg-4: hsl(0, 0%, 45%);
+	--sk-fg-accent: hsl(12, 93%, 43%);
+
+	--shiki-color-text: var(--sk-fg-2);
+	--shiki-token-constant: var(--shiki-color-text);
+	--shiki-token-string: hsl(30, 100%, 30%);
+	--shiki-token-comment: var(--sk-fg-4);
+	--shiki-token-keyword: hsl(204, 100%, 38%);
+	--shiki-token-parameter: var(--shiki-color-text);
+	--shiki-token-function: var(--sk-fg-accent);
+	--shiki-token-string-expression: var(--shiki-token-string);
+	--shiki-token-punctuation: var(--shiki-color-text);
+	--shiki-token-link: var(--shiki-token-keyword);
+
+	/* Background colours */
+	--sk-bg-1: hsl(0, 0%, 100%);
+	--sk-bg-2: hsl(0, 0%, 100%);
+	--sk-bg-3: hsl(0, 0%, 99%);
+	--sk-bg-4: hsl(0, 0%, 95%);
+	--sk-bg-5: hsl(0, 0%, 92%);
+	--sk-bg-accent: var(--sk-fg-accent);
+	--sk-bg-selection: hsla(204, 100%, 63%, 0.3);
+
+	/* Border colors */
+	--sk-border-light: hsl(0, 0%, 95%);
+	--sk-border-medium: hsl(0, 0%, 92%);
+	--sk-border-dark: hsl(0, 0%, 86%);
```

Comment:
are these the right names? in dark mode it's the opposite of what they say. Going with the -1/2/3 of the bg/fg seems more straightforward

Reply:
> in dark mode it's the opposite of what they say.

No it's not, check again. This isn't the same as `-1`, `-2`, `-3` etc because for foreground/background colours you're coming from the 'edges' — `--sk-fg-1` is the darkest in light mode and the lightest in dark mode, and vice versa for `bg` (or should be, except that we currently do things in a confusing way which will be fixed in the follow-up PR).

For borders on the other hand, lighter should mean lighter, since it's used as a highlight for the top and left borders. (In fact maybe it should be `--sk-border`, `--sk-border-highlight` and `--sk-border-shadow` or something)